styles.css

```
@@ -8,6 +8,60 @@
   src: url("assets/fonts/Inter-Light.ttf");
 }
 
+.hide {
+  display: none;
+}
+
+.close-btn {
+  height: 12px;
+  width: 12px;
+  align-self: flex-end;
+  justify-self: flex-end;
+  margin-left: 90%;
+  margin-top: 10%;
+}
+
+ul {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  margin: 0;
+  padding-left: 0;
+  width: 60vw;
+  height: 30px;
+  flex-wrap: wrap;
+}
+
+li {
+  list-style: none;
+}
+
+#menu {
+  height: 100vh;
+  width: 100vw;
+  background-color: white;
+}
```

```suggestion
#menu {
    height: 100vh;
    width: 100vw;
    background-color: white;
    position: fixed;
    overflow-y: hidden;
    top: 0;
    left: 0;
}
```
[OPTIONAL]
- [ ] You might consider making the mobile menu fixed and preventing it from scrolling with the page.